cmd/kubeadm/app/preflight/checks.go

```diff
@@ -850,6 +851,31 @@ func getEtcdVersionResponse(client *http.Client, url string, target interface{})
 	return err
 }
 
+// ResolveCheck tests for potential issues related to the system resolver configuration
+type ResolveCheck struct{}
+
+// Name returns label for ResolveCheck
+func (ResolveCheck) Name() string {
+	return "Resolve"
+}
+
+// Check validates the system resolver configuration
+func (ResolveCheck) Check() (warnings, errors []error) {
+	warnings = []error{}
```

Comment:
I modified this patch to remove the glog business since that was added somewhere after 1.10